README.md

```diff
@@ -8,10 +8,13 @@ This is based on an open source autosizing text component for Hmos.
 Inspired by [erchenger/SizeAdjustingTextView](https://github.com/erchenger/SizeAdjustingTextView)
 
 ## Feature
-This is based on autosizing text component for Hmos.
+SizeAdjustingTextView is a library that allows user to input text of any length of characters which will auto resize the text within the custom component.
+Resizing multiple lines in the text component to keep up the changes.
+
+<img src="screenshots/gif.gif" width="500">
```

Comment:
Please make sure the gif added is clearly visible to the user. Current gif needs to be changed